README.md

```diff
@@ -69,46 +83,36 @@ rLogin.on("chainChanged", (chainId) => {
 })
 ```
 
-## Optional parameters
+#### About Metamask and chain ID change
 
-### autoRefreshOnNetworkChange
+Changing the chain may result in the wallet refreshing the page. If this is not the desired effect, and the wallet supports EIP-1198 `chainChanged`, you can use:
 
-Changing the chain may result in the wallet refreshing the page. If this is not the desired effect, and the wallet supports this parameter, you can pass this in.
-
-```js
-const rLogin = new window.RLogin.default({
-    autoRefreshOnNetworkChange: false,
-})
+```javascript
+window.ethereum.autoRefreshOnNetworkChange = false
```

Comment:
Does this use the injected web3 provider from the wallet? If so, Metamask is planning on not injecting it in the near future which is why I passed the parameter into rLogin. [reference](https://medium.com/metamask/no-longer-injecting-web3-js-4a899ad6e59e)